src/main/java/io/swagger/configuration/APIServiceConfiguration.java

```diff
@@ -1,14 +1,7 @@
 package io.swagger.configuration;
 
-import io.swagger.entity.media.MediaInfo;
-import io.swagger.repository.MediaRepository;
-import io.swagger.service.MediaService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import javax.annotation.PostConstruct;
-
 @Configuration
 public class APIServiceConfiguration {
```

Comment:
Since introducing me to the @Service annotation, we can probably delete this file in a future PR.